### DISCUSSION TITLE
How to get started?

### DISCUSSION THREAD

**Discussion by Tushar-2003** [keploy/keploy#181] [Q&A] [answered]
I would like to contribute to the project, where can i start.

**About me**
- I have used keploy before(fellowship).
- Basic idea about Go.
- can do documentation.

**Comment by indevi** [comment, marked as answer]
Hi @Tushar-2003 Thanks for your interest. Here are few [good-first-issues ](https://github.com/keploy/keploy/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22). Feel free to explore them and pick an issue that you would like to work on. 
As you have also shown interest  in documentation. You can take a look at [this](https://github.com/keploy/docs)

In case of queries , join us on Slack :)

**Reply by Tushar-2003** [reply]
Thanks @ShivamTyagi12345
